src/app/plan/duck/actions.ts

```diff
@@ -51,9 +49,29 @@ export const PlanActionTypes = {
   PLAN_POLL_STOP: 'PLAN_POLL_STOP',
   RESET_CURRENT_PLAN: 'RESET_CURRENT_PLAN',
   SET_CURRENT_PLAN: 'SET_CURRENT_PLAN',
-  SET_LOCKED_PLAN: 'SET_LOCKED_PLAN'
+  SET_LOCKED_PLAN: 'SET_LOCKED_PLAN',
+  SET_PLAN_ADD_EDIT_STATUS: 'SET_PLAN_ADD_EDIT_STATUS',
+  WATCH_PLAN_ADD_EDIT_STATUS: 'SET_PLAN_ADD_EDIT_STATUS',
```

Comment:
This is a copy paste error using the same value as SET PLAN

Reply:
Nice catch - updated 👍